.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Runing End to End tests
```

Comment:
I think the job here should be integrated into the base workflow and not live in its own file. That way, we can continue working with the code that was already built and just make this about running the e2e tests. With this setup (because they are different workflows) this results in us running the full build twice—once in the standard workflow, once here. Also, it doesn't really make sense to run the e2e tests if the build fails earlier in the process...

Reply:
hey @ibacher i have merged the two workflows but starting docker on windows is real pain.  I have introduced conditional execution so that docker builds only run on ubuntu and not windows.  
I what i have done ok??